app.rb

```diff
@@ -68,8 +68,8 @@ def version_from_git(dir)
   cluster = CLUSTERS[@id]
   if cluster.nil?
     raise Sinatra::NotFound
-  else
-    @ganglia = Ganglia.new(cluster).send(@time)
+  elsif cluster.custom_config.key?(:ganglia)
```

Comment:
This should probably `raise Sinatra::NotFound` if `cluster.nil?` or `! cluster.custom_config.key?(:ganglia)`.

Reply:
Fixed